fe/src/main/java/org/apache/doris/analysis/CreateMaterializedViewStmt.java

```diff
@@ -188,6 +189,7 @@ else if (functionChild0 instanceof CastExpr
                                                         + "Error function: " + functionCallExpr.toSqlImpl());
                 }
                 meetAggregate = true;
+                mvKeysType = KeysType.AGG_KEYS;
```

Comment:
Why setting `mvKeysType` again since you already set it before?
Do we allowed user create mv like `select k1, k2, sum(v1)` without `group by` clause?

Reply:
This query `select k1, k2, sum(v1) from table` will be forbidden by analyzer.